src/pb.rs

```diff
@@ -134,6 +148,70 @@ impl ProgressBar {
         }
     }
 
+    /// Set message to display in the prefix, call with "" to stop printing a message.
+    ///
+    /// # Examples
+    /// ```ignore
+    /// let mut pb = ProgressBar::new(20);
+    /// 
+    /// for x in 0..20 {
+    ///    match x {
+    ///       0 => pb.message("Doing 1st Quarter"),
+    ///       5 => pb.message("Doing 2nd Quarter"),
+    ///       10 => pb.message("Doing 3rd Quarter"),
+    ///       15 => pb.message("Doing 4th Quarter"),
+    ///    }
+    ///    pb.inc().
+    /// }
+    /// 
+    /// ```
+    pub fn message(&mut self, message: &str) {
+        self.message = message.to_owned()
+    }
+
+    /// Set tick format for the progressBar, default is \\|/-
+    /// 
+    /// Format is not limited to 4 characters, any string can
+    /// be used as a tick format (the tick will successively
+    /// take the value of each char but won't loop backwards).
+    /// 
+    ///
+    /// # Examples
+    /// ```ignore
+    /// let mut pb = ProgressBar::new(...);
+    /// pb.tick_format("▀▐▄▌")
+    /// ```
+    pub fn tick_format(&mut self, tick_fmt: &str) {
+        if tick_fmt != TICK_FORMAT {self.show_tick = true;};
```

Comment:
Can you fix this `if` to follow [rustfmt](https://github.com/rust-lang-nursery/rustfmt)?